protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/processinstance/ProcessInstanceCreationRecord.java

```diff
@@ -37,20 +37,25 @@ public final class ProcessInstanceCreationRecord extends UnifiedRecordValue
   private final ArrayProperty<StringValue> fetchVariablesProperty =
       new ArrayProperty<>("fetchVariables", new StringValue());
 
+  private final ArrayProperty<ProcessInstanceCreationStartInstruction> startInstructionsProperty =
+      new ArrayProperty<>("startInstructions", new ProcessInstanceCreationStartInstruction());
```

Comment:
🔧 A getter for this property is missing. We can also add it later when we need it.

Reply:
I modeled it after `fetchVariables` There is a sort of getter for both, which returns the property. A property for the converted value can be added when we feel it becomes necessary.